lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -95,6 +95,7 @@ class ProgressTabSerializer(VerifiedModeSerializerMixin):
     """
     Serializer for progress tab
     """
+    username = serializers.CharField()
```

Comment:
Oh also - this and the coaching logic etc should probably have some tests yeah?

Reply:
added a test